mentors/Makefile

```diff
@@ -31,7 +31,7 @@ $(WATSON_CREDENTIALS):
 # Removes all mentor files from the local file system
 .PHONY clean:
 clean:
-	rm -rf data/recordings/*
+	rm -rf $(MENTOR)/build/*
```

Comment:
Is this an acceptable way to handle cleaning of data? Should I name it clean-mentor?

Reply:
What's the advantage of having recordings under `data` instead of `build`?

Reply:
Yeah I think that looks right. Right now there's no other clean operation right?

Reply:
More common I think is to just delete the build directory altogether as opposed to deleting its contents

Reply:
Nope, this is the only one. 

At this point I would assume that people will be working with the data one mentor at a time.

Reply:
ah, i see what you're saying. I do think we want to phase out that `MENTOR` var. Can it be this instead?

```
cd $(MENTOR_DIR) && \
for m in */; do rm -rf $${m}/build; done
```

?

Reply:
Yeah, should we specify the mentor here or should we just blow everything away? I think it makes more sense to just clean out one mentor at a time but I don't know if that is expected behavior for this.

Reply:
usually when people run clean, they want the whole project clean. If you want to also be able to clean just one how about this:

```
.PHONY: mentor/%/clean
clean/%:
	@rm -rf "$*/build"

.PHONY: clean
clean:
	@for m in */; do $(MAKE) clean/$${m}; done
```

..which would allow for `make clean/clint`

We also have a separate issue, which is that the mentors should be in data folder (e.g. `/.mentors/data/clint`) to separate them from `src` and other none mentor-data, but let's hold off on that because will require touching a lot of other code